tests/test_stl.py

```diff
@@ -283,3 +283,11 @@ def test_issue_1561():
     bar.list = [m.Issue1561Inner("bar")]
     bar.list
     assert bar.list[0].data == "bar"
+
+
+def test_return_vector_bool_raw_ptr():
+    while True:
+        v = m.return_vector_bool_raw_ptr()
+        assert isinstance(v, list)
+        assert len(v) == 4513
+        break  # Comment out for manual leak checking (use `top` command).
```

Comment:
Kind of odd to have in a test? Maybe 
```suggestion
    # Add while True: for manual leak checking
    v = m.return_vector_bool_raw_ptr()
    assert isinstance(v, list)
    assert len(v) == 4513
```

Reply:
Done. Thanks Henry!

Side remark & question:
* I have a lot of those on the smart_holder branch, and need to turn them on-off quite a bit when going into the smart_holder code.
* Do you happen to be aware of a pytest feature or plugin that would automate the fool-proof top-like leak check with adding a certain flag?

I've been thinking of this as a potential Google Summer of Code project if it doesn't exist already. Roughly: run decorated test functions in a loop while monitoring the equivalent of top's RES from within the same Python process, use some simple heuristics to decide "this could be a leak" or "no this doesn't look like a leak after running for N seconds".